thirdparty/zlib/CMakeLists.txt

```diff
@@ -36,11 +40,15 @@ set(ZLIB_VER "1.2.11")
 set(ZLIB_MD5 "1c9f62f0778697a09d36121ead88e08e")
 ExternalProject_Add(
     ${PROJECT_NAME}
-    DOWNLOAD_DIR ${KO_DOWNLOAD_DIR}
+    DOWNLOAD_DIR @KO_DOWNLOAD_DIR@
     URL http://gentoo.osuosl.org/distfiles/zlib-${ZLIB_VER}.tar.gz
     URL_MD5 ${ZLIB_MD5}
     BUILD_IN_SOURCE 1
     ${optional_cfg_cmd}
     BUILD_COMMAND ${BUILD_CMD}
     INSTALL_COMMAND ${INSTALL_CMD}
 )
+
+set(ZLIB_INCLUDE_DIR "${SOURCE_DIR}/include" PARENT_SCOPE)
+set(ZLIB_LIB_DIR "${SOURCE_DIR}/lib" PARENT_SCOPE)
+set(ZLIB_STATIC "${SOURCE_DIR}/lib/libz.${CMAKE_STATIC_LIBRARY_SUFFIX}" PARENT_SCOPE)
```

Comment:
Probably also needs `ZLIB`

The idea being not to do all this stuff quite so excessively:

https://github.com/koreader/koreader-base/blob/6582561d074cf332ba1e9df9a459fe09b0ee4b26/Makefile.third#L130-L157